content/general/concepts/http/http.md

```diff
@@ -0,0 +1,107 @@
+---
+Title: 'HTTP'
+Description: 'HTTP is used for fetching HTML documents and other Web-based resources.'
```

Comment:
I don't believe web should be capitalized here

Reply:
Fair enough. It's been around for a while. 😄